week-1-practise/level-1/9-function-output-2.js

```diff
@@ -1,14 +1,20 @@
 function s(w1, w2) {
-    return w1.concat(w2);
+  return w1.concat(w2);
 }
 
-var result = s('code', 'your');
+var result = s("code", "your");
```

Comment:
It looks like you changed the single quotes to double quotes. Single quotes are also fine. In JS you can use either of them to define a string.

Reply:
Yes, you are right. I use double quotes but i will use both of them.